Source/DataSources/BillboardVisualizer.js

```diff
@@ -50,6 +50,7 @@ define([
     var EntityData = function(entity) {
         this.entity = entity;
         this.billboard = undefined;
+        this.textureValue = undefined;
```

Comment:
I totally agree that this isn't ideal, but I think in the short term it's the best we can do.  Most of the time it's just going to be a string.  We can revisit this after KML.